hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/test/java/org/apache/hadoop/yarn/server/federation/store/impl/TestZookeeperFederationStateStore.java

```diff
@@ -96,6 +96,7 @@ public void before() throws IOException, YarnException {
       Configuration conf = new YarnConfiguration();
       conf.set(CommonConfigurationKeys.ZK_ADDRESS, connectString);
       conf.setInt(YarnConfiguration.FEDERATION_STATESTORE_MAX_APPLICATIONS, 10);
+      conf.setInt(YarnConfiguration.ZK_APPID_NODE_SPLIT_INDEX, 1);
```

Comment:
Can we have better coverage?

Reply:
I will write this configuration into yarn-site.xml for testing.